documentation/asciidoc/topics/yaml/cache_service.yaml

```
@@ -3,6 +3,9 @@ kind: Infinispan
 metadata:
   # Names the cluster.
   name: {example_crd_name}
+  annotations:
+    # Creates a ServiceMonitor for the cluster.
+    infinispan.org/monitoring: 'false'
```

`false` means disable ServiceMonitor
```suggestion
    infinispan.org/monitoring: 'true'
```